jvm/src/test/scala/org/scalacheck/GenSpecification.scala

```diff
@@ -54,10 +54,15 @@ object GenSpecification extends Properties("Gen") {
     forAll(g) { n => true }
   }
 
-  property("frequency 3") = forAll(choose(0,100000)) { n =>
+  property("frequency 3") = forAll(choose(1,100000)) { n =>
     forAll(frequency(List.fill(n)((1,const(0))): _*)) { _ == 0 }
   }
 
+  property("frequency 4") =
```

Comment:
This one could be called "frequency 0" since it's for zero-arguments given.

Good idea to capture this condition with a test!

Reply:
May as well be exhaustive. I don't know if `Gen.frequency` is useful without any arguments though. I think that the method should require at least one argument, but that may result in some binary incompatibility.